testproject/Assets/Tests/Runtime/NetworkSceneManagerTests.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace TestProject.RuntimeTests
 {
-    public class NetworkSceneManagerTests: BaseMultiInstanceTest
+    public class NetworkSceneManagerTests : BaseMultiInstanceTest
```

Comment:
Just nitpicking, this file doesn't have any actual changes, so for a big review like this I think it would be in your best interest to omit these types of changes

Reply:
dotnet-format did this :/... it changed a lot of files when I ran standards.py --fix and I didn't look through to see whether all of them were mine before I checked in... Sorry about that... but at the same time... probably worth getting develop to a point where standards.py --fix does nothing to it? Maybe in a different PR though...

Reply:
Yeah, my point is mainly about keeping PRs focused, this file is a distraction and makes the PR look bigger than it actually is, so it would help with review traction to do a quick audit for these kinds of things before putting it up

Reply:
we sometimes get to that "`./standards.py --fix` does nothing" point but people often do not run `./standards.py --fix` locally before they push their changes (even though you could do `./standards.py --hook` for it) and we get these whitespace updates whenever somebody else does and fixes things.